tags/workarea.tag.html

```diff
@@ -86,73 +138,101 @@
             }
         });
 
-        this.on('mount', function () {
+        /**
+         * Updates workarea when self.update() is called or when workarea changes
+         */
+        this.on('update', function() {
+            myCanvas.clear();
+            drawOnCanvas();
+        })
+
+        /**
+         * Removes listeners when unmounting tag to prevent multiple instances of keyup and click from firing
+         */
+        this.on('unmount', function() {
```

Comment:
Will it detach hot keys for tool selection, open/save, import/export etc.? Let us know if it is tested?

Reply:
This is tested. It only removes the 'click' and 'keyup' events during 'unmount' or when we switch between images.